ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main-presubmits.yaml

```diff
@@ -138,7 +138,7 @@ presubmits:
     name: pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-tempest
     optional: true
     rerun_command: /test cinder-operator-build-deploy-tempest
-    skip_if_only_changed: ^renovate.json|^README.md
+    skip_if_only_changed: ^(renovate.json|.*\.md|tests/.*|OWNERS.*|LICENSE.*)$
```

Comment:
I'm not familiar with this stuff, but the rerun_command suggests this job just runs tempest. If that's true, and it doesn't execute the functional tests, then maybe the skip list could include "tests/functional/.*" (like the one on L78).

Reply:
The proposed pattern includes `tests/.*` so that is already covered. The current pattern also avoids running tempest tests when change is limited to `tests/kuttle/.*` because the directory does not really contain anything used by tempest job afaik.

Reply:
Whoops, I need to clean my glasses. Of course, skipping tests/.* will certainly include tests/functional.